source/components/organisms/Step/StepFooter/StepFooter.tsx

```diff
@@ -136,17 +144,17 @@ const StepFooter: React.FC<Props> = ({
   };
 
   const buttons = actions.map((action, index) => (
-    <Flex key={`${index}-${action.label}`}>
-      <Button
-        onClick={actionMap(action)}
-        colorSchema={action.color}
-        disabled={isLoading || (action.hasCondition && checkCondition(action.conditionalOn))}
-        z={0}
-      >
-        <Text>{action.label}</Text>
-      </Button>
-    </Flex>
-  ));
+        <Flex key={`${index}-${action.label}`}>
```

Comment:
It isn't best practices to use **index** in the key property since that could possibly affect the React performance.

Reply:
I'm not sure if this would break things, and I'm not sure if I should change it, as I don't see any other simple way of making the key unique.

Reply:
Agree, looking at the code a possible value could be `${action.type}-${action.label}` as it should be sufficiently unique.